que_.py

```diff
@@ -0,0 +1,26 @@
+from doubly_linkedlist import Dll
+
+
+class Queue:
+    def __init__(self, values=None):
+        self._storage = Dll(values)
+
+    def enqueue(self, val):
+        self._storage.append(val)
+
+    def dequeue(self):
+        try:
+            return self._storage.pop()
+        except ValueError:
+            raise ValueError("The queue is empty.")
+
+    def peek(self):
+        if len(self._storage) >= 2:
+            return self._storage.head.next.value
+        return None
```

Comment:
Peak has nothing todo with .next.  It's the value that would come off with the next dequeue call without actually dequeing it.

Reply:
ohhh okay. i miss understood